plutus-core/prelude/PlutusPrelude.hs

```diff
@@ -86,6 +85,7 @@ module PlutusPrelude
     , printPretty
     -- * Text
     , showText
+    , module Data.Default.Class
```

Comment:
I reexport the Default class from PlutusPrelude, because a lot of the codebase requires 'def' now. I am not sure if this is the best place though, and I want to hear your opinions @effectfully @kwxm .

Other ideas I had was to:
- export somewhere from PlutusCore exposed module
- do not export at all, and let the downstream code depend on 'data-default-class' cabal package.

Reply:
I don't have any particular opinion on this.  I always find it a bit difficult to work out where imports are coming from anyway. Recently I've been trying to list imports explicitly, but that's impractical with something big like `PlutusPrelude` or `PlutusCore`.

Reply:
> I don't have any particular opinion on this.

Me neither. I don't like that `def` looks like a local name rather than a global one, but it's pretty established, so I don't mind it either. I think reexposting it from here is fine.

I'd export it as `Default (..)` rather than `module Data.Default.Class`, though. Just for clarity.